Makefile

```diff
@@ -138,13 +138,17 @@ doc-setup: client
 	rm -Rf doc/.sphinx/.doctrees
 
 .PHONY: doc
-doc: doc-setup doc-incremental
+doc: doc-setup doc-incremental doc-objects
 
 .PHONY: doc-incremental
 doc-incremental:
 	@echo "Build the documentation"
 	. $(SPHINXENV) ; LOCAL_SPHINX_BUILD=True sphinx-build -c doc/ -b dirhtml doc/ doc/html/ -d doc/.sphinx/.doctrees -w doc/.sphinx/warnings.txt
 
+.PHONY: doc-objects
+doc-objects:
+	. $(SPHINXENV); cd doc/html; python3 -m sphinx.ext.intersphinx 'objects.inv' > objects.inv.txt
```

Comment:
What format is this new file in @ru-fu ?

Also, is it possible to add a comment here explaining what its intended to be used for? I.e to be consumed by the UI?

Reply:
It's a text file.
And do you mean a comment in the Makefile? I have an explanation in this PR.

Reply:
Yeah in the makefile thanks.

Are the UI team able to parse the text file?

Reply:
@edlerd suggested this change, so I would assume so. ;)

Reply:
> Are the UI team able to parse the text file?

We have to write a parser for it, but that should be trivial. So: Yes.

Reply:
@edlerd you've seen the text format and happy with it?

Reply:
Excellent thanks @edlerd